src/main/java/uk/gov/hmcts/reform/slc/logging/AppInsights.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.slc.logging;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.telemetry.Duration;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.logging.appinsights.AbstractAppInsights;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class AppInsights extends AbstractAppInsights {
+
+    public AppInsights(TelemetryClient telemetryClient) {
+        super(telemetryClient);
+    }
+
+    // DEPENDENCIES
+
+    public void trackMessageReceivedFromServiceBus(long durationInMillis, boolean success) {
```

Comment:
When this method is called you convert `Duration` to millis and here you convert back to `Duration`.
Just pass `Duration`.

Reply:
That's a different duration - the one in microsoft's appsinsights

Reply:
:D

Reply:
true. there are lot of repetitive actions in here really. but left it out of consideration for this PR